lib/game.rb

```diff
@@ -33,19 +33,20 @@ def start
 
       player_1_submarine_coords = gets.chomp.upcase.split
 
-      until player_1.board.valid_placement?(@player_1.fleet[1], player_1_submarine_coords) == true
+      until @player_1.board.valid_placement?(@player_1.fleet[1], player_1_submarine_coords) == true
         p "Those are invalid coordinates. Please try again:"
-        player_1_submarine_coords = gets.chomp.upcase.split
+        @player_1_submarine_coords = gets.chomp.upcase.split
       end
 
       @player_1.board.place(@player_1.fleet[1],player_1_submarine_coords)
 
-      p '============COMPUTER BOARD=============='
-      p @computer.board.render
-      p '=============PLAYER BOARD==============='
-      p @player_1.board.render_player(true)
+      print '============COMPUTER BOARD=============='
+      print @computer.board.render
```

Comment:
line 43 & 45 need to be "p" to fix bug.

on the first round, the "1 2 3 4" are not printing on the next line after "==COMPUTER BOARD==", the integers are printing on the same line with print command.